src/components/Navigator/NavigatorCard.vue

```diff
@@ -1060,7 +1060,7 @@ $navigator-card-vertical-spacing: 8px !default;
   @include breakpoint(medium, nav) {
     --nav-filter-horizontal-padding: 20px;
     border: none;
-    padding-top: 0px;
+    padding-top: 10px;
     padding-bottom: 10px;
```

Comment:
Can we simplify these ones in:

```suggestion
    padding: 10px 0;
```